debian/debian/ix-vendor.service

```diff
@@ -1,14 +1,11 @@
 [Unit]
 # vendor_service.py must run after network routes are set up.
-After=ix-postinit.service
+After=ix-netif.service
```

Comment:
I think we should keep this to `ix-postinit` as that means the system is ready at this point and then we are calling some vendor script which can be anything and can try to do anything which might/might not work based on how far along middleware is ready at that point.
We should change `WantedBy` to `truenas.target` instead - that could probably be causing the dependency issue